### PR TITLE
some mesh tests

### PR DESCRIPTION
**Pull request by jeffj6123**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/mesh/azext_mesh/tests/latest/test_mesh_commands.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        # Test show
+        self.cmd('az mesh service-replica show --resource-group {rg} --app-name {app_name} --service-name {service_name} --replica-name {replica_name}', checks=[
+            self.exists('codePackages'),
+            self.exists('networkRefs'),
+            self.exists('osType'),
+            self.exists('replicaName'),
+        ])
+
+        self.cmd('az mesh app delete -g {rg} --name {app_name} --yes')
+
```
Comment (linggengmsft) [inline src/mesh/azext_mesh/tests/latest/test_mesh_commands.py]: Could you add the test for container log as well?

File: src/mesh/azext_mesh/tests/latest/recordings/test_service_replica_commands.yaml [hunk 2]
```diff
@@ -0,0 +1,548 @@
+interactions:
```
Comment (linggengmsft) [inline src/mesh/azext_mesh/tests/latest/recordings/test_service_replica_commands.yaml]: Add the instruction how to run tests for Mesh in README page as a reference for future contributions from others.

**Comment by linggengmsft** [comment]
@jeffj6123 the CI test is failing

File: src/mesh/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.9.1 (2018-07-26)
```
Comment (linggengmsft) [inline src/mesh/HISTORY.rst]: Update the date

File: src/mesh/README.rst [hunk 4]
```diff
@@ -1,6 +1,13 @@
 Microsoft Azure CLI 'mesh' Command Module
 ==============================================================
 Official doc https://docs.microsoft.com/en-us/azure/service-fabric-mesh/
+
+Run tests using
+python tests/latest/test_mesh_commands.py
+Delete the recordings file to run live tests instead of through vcr.
```
Comment (linggengmsft) [inline src/mesh/README.rst]: How to delete the recording files?

File: src/mesh/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.9.2 (2018-07-26)
```
Comment (linggengmsft) [inline src/mesh/HISTORY.rst]: This is still not fixed

File: src/mesh/README.rst [hunk 6]
```diff
@@ -1,6 +1,14 @@
 Microsoft Azure CLI 'mesh' Command Module
 ==============================================================
 Official doc https://docs.microsoft.com/en-us/azure/service-fabric-mesh/
+
+Run tests using
+python tests/latest/test_mesh_commands.py
+Delete the recordings file to run live tests instead of through vcr by deleting
```
Comment (linggengmsft) [inline src/mesh/README.rst]: So this line can be deleted as well?

Comment (jeffj6123) [reply src/mesh/README.rst]: I reworded it to make more sense.

**Pull request merged**